src/v2/guide/instance.md

```diff
@@ -64,25 +86,25 @@ var vm = new Vue({
   data: data
 })
 
-vm.$data === data // -> true
-vm.$el === document.getElementById('example') // -> true
+vm.$data === data // => true
+vm.$el === document.getElementById('example') // => true
 
 // $watch is an instance method
-vm.$watch('a', function (newVal, oldVal) {
-  // this callback will be called when `vm.a` changes
+vm.$watch('a', function (newValue, oldValue) {
+  // This callback will be called when `vm.a` changes
 })
 ```
 
-<p class="tip">Don't use [arrow functions](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/Arrow_functions) on an instance property or callback (e.g. `vm.$watch('a', newVal => this.myMethod())`). As arrow functions are bound to the parent context, `this` will not be the Vue instance as you'd expect and `this.myMethod` will be undefined.</p>
-
-Consult the [API reference](../api/#Instance-Properties) for the full list of instance properties and methods.
+In the future, you can consult the [API reference](../api/#Instance-Properties) for a full list of instance properties and methods.
 
 ## Instance Lifecycle Hooks
 
-Each Vue instance goes through a series of initialization steps when it is created - for example, it needs to set up data observation, compile the template, mount the instance to the DOM, and update the DOM when data changes. Along the way, it will also invoke some **lifecycle hooks**, which give us the opportunity to execute custom logic. For example, the [`created`](../api/#created) hook is called after the instance is created:
+Each Vue instance goes through a series of initialization steps when it's created - for example, it needs to set up data observation, compile the template, mount the instance to the DOM, and update the DOM when data changes. Along the way, it also runs functions called **lifecycle hooks**, giving users the opportunity to add their own code at specific stages.
+
+For example, the [`created`](../api/#created) hook can be used to run code after an instance is created:
```

Comment:
In this case, `can be used to run code after` does actually say what I mean. What meaning does it have for you?